packages/design-system/package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@appsmithorg/design-system",
-  "version": "1.0.45",
-  "description": "This is the package for the Appsmith design system components",
+  "name": "@appsmithorg/design-system-v2",
+  "version": "2.0.0-alpha.0",
+  "description": "",
```

Comment:
"description": "This is the package for the design system that powers the Appsmith platform",